\chapter{Introduction}

	\section{Gene regulation enters biology spotlight}

Vertebrates are a subphylum of Chordata characterized by the presence of a backbone or spinal column, and as multicellular organisms, they represent mosaics of cell types. Understanding what mechanisms and regulators control cellular fate and lead to the observed variety of morphologies and physiologies has been one of the longstanding questions in biology. One of the first hypothesis to explain this phenomenon was that fate depends on the presence of a given set of genes, thus, non-essential genes for a cell type would have to be lost during differentiation. However, this hypothesis was discarded in light of the results obtained by nuclei transplantation experiments in frog eggs. In these experiments the nucleus of a differentiated cell from the intestine epithelium was injected into enucleated eggs giving rise to normal adult frogs. The fact that nuclei of differentiated cells are able to resume normal development indicates that those cells have all the genes required to form different specialized cell types and that non-expressed genes are not permanently inactivated. Moreover, the results of these experiments suggested a role of cytoplasmic components in the regulation of gene expression (Gurdon, 1968). This first indication that the genome was the major commonality between different cell types of an organism was confirmed through sequencing of whole genomes, and it applies to almost all healthy somatic cell types with clear exceptions, such as, cells from the immune system which genomes undergo recombination of specific loci (Market and Papavasiliou 2003). In consequence, it was concluded that if there are no major differences in terms of gene content, what generates the myriad of cells types is the precise regulation of genes.\\

Initial understanding of gene regulation was gained through studies of enzymatic induction in bacteria. Importantly, those studies demonstrated the existence of regulatory sequences and gene products that could impact the expression of other genes (Pardee et al. 1958; Jacob and Monod, 1961). In contrast to bacteria, eukaryotic cells are more complex structures in which the genome is confined to a nucleus and packed by histones constituting nucleosomes that arrange into chromatin fibers. Therefore, gene expression in vertebrates and other eukaryotes is influenced by more layers of regulation than in bacteria, and these regulatory events occur both in the nucleus and in the cytoplasm. In this work I have focused on the first regulatory layer of gene expression that is transcriptional regulation, and in particular, on mechanisms and regulators controlling initiation and maintenance of transcription.\\

	\section{Transcriptional regulation}
Gene transcription in eukaryotes can be orchestrated by three protein complexes named RNA polymerase I, II and III (Pol I, Pol II and Pol III) that undergo the main steps of initiation, elongation and termination to synthesize different classes of RNAs. Detailed structural analyses of these protein complexes have determined that, in contrast to their mechanisms of elongation that are conserved, the mechanisms that trigger initiation of transcription differ. Substantial differences are mainly observed by comparing the structures of Pol I and Pol II, and it has been hypothesized that these differences may explain the evolution of three polymerases in eukaryotes to regulate different target genes (Engel et al. 2018). While Pol I and Pol III transcribe genes of noncoding RNAs, most of the RNAs in a eukaryotic cell, including messenger RNAs (mRNAs), are transcribed by Pol II. Comparative analysis of the yeast and human Pol II structures during transcriptional initiation have revealed that these complexes are conserved in eukaryotes (Hantsche and Cramer 2017).\\

To initiate transcription, polymerases have to interact with basal, or general, transcription factors that recruit them to flanking sequences of the transcription start sites (TSSs), known as core promoters, to assemble the preinitiation complex. In the case of Pol II, the factors with which it interacts are denoted as transcription factors for Pol II (TFII) and their loading occurs in sequential order. First, TFIID recognizes and binds distinctive sequence motifs of the core promoter and in combination with TFIIB and TFIIA, it induces DNA bending which favors the loading of Pol II pre-bound to TFIIF and TFIIE. After formation of the complex, the factors mediate unwinding of DNA that enables Pol II to scan the sequence and find the TSS to initiate transcription. When the newly synthesized RNA reaches a critical length and the Pol II carboxy-terminal domain (CTD) has been phosphorylated by TFIIH, the initiation complex dissociates and Pol II strengths its interaction with DNA to start the loading of the elongation complex (Alberts et al. 2007; Engel et al. 2018). The initiation complex also comprises the co-activator complex Mediator that stabilizes the interactions between Pol II and the basal transcription factors. Furthermore, it has been determined that Mediator has additional functions during transcriptional regulation. For instance, based on current evidence it has been proposed that Mediator first interacts with distal regulatory sequences to be next used as a bridge between those sequences and core promoters, where it facilitates the assembly of the initiation complex and later its disassembly by stimulating TFIIH kinase activity (Jeronimo and Robert 2017). Finally, Mediator participates in alleviating Pol II pausing that occurs in most promoters after the initial extension of RNA to tenths of nucleotides (Jeronimo and Robert 2017; Mayer et al. 2017).\\

As mentioned, basal transcription factors bind to core promoters in order to initiate transcription. Nevertheless, transcription mediated solely by core promoters is not efficient and additional regulatory sequences, known as \textit{cis}-regulatory elements, have to contribute increasing the levels of basal transcription (Wittkopp and Kalay 2012). The main regulatory elements and their mechanisms of action are described in the following sections.\\

		\subsection{Promoters}

In general, promoters are defined as the sequences surrounding the TSSs including the core promoter and its upstream sequence (proximal promoter). Based on the presence of a single or multiple TSSs, promoters can be classified into focused or dispersed promoters, respectively. The selection of a specific TSS within dispersed promoters is partially mediated by nucleosome positioning (Forrest et al. 2014).\\

Core promoters frequently contain sequence motifs that are recognized by the basal transcription factors. These motifs include the TATA box that is located $\sim$30 bp upstream of the TSS and conserved throughout eukaryotes, the Initiator motif that overlaps TSSs and TFIIB recognition elements, among others (Decker and Hinton 2013; Haberle and Stark 2018). It should be noted that sequence motifs are not present in all core promoters, for instance, only a small percentage of mammalian core promoters contain the TATA box (Gershenzon and Ioshikhes 2005) and the presence of several motifs within a single core promoter might be contra productive in the disassembly of the initiation complex. This observation supports a model in which different combinations of motifs influence the factors that bind to the core region, conferring a similar regulation than the one mediated by differential usage of $\sigma$ factors in bacteria (Decker and Hinton 2013). Also, sequence analyses of the promoters of housekeeping genes and genes involved in vertebrate development have depicted CpG islands as an additional characteristic of these promoters (Deaton and Bird 2011).\\

At the chromatin level, active promoters are distinguished by high abundance of two chromatin marks, tri-methylation of lysine 4 on histone H3 (H3K4me3) and acetylation of lysine 27 on histone H3 (H3K27ac) (Barski et al. 2007). Notably, promoters of genes that play important functions in development are often found in a poised state in pluripotent cells. This state is delineated by bivalent enrichment on the active mark H3K4me3 and the repressive mark tri-methylation of lysine 27 on histone H3 (H3K27me3) (Azuara et al. 2006; Bernstein et al. 2006). Given the lack of a consensus sequence at core promoters, efforts to identify active promoters have relied on mapping 5’ ends of RNA and nascent RNAs, as well as, on mapping active histone marks by chromatin immunoprecipitation followed by sequencing (ChIP-seq) (Forrest et al. 2014; Danko et al. 2015; Guenther et al. 2007). Although there is high correlation between the presence of certain chromatin marks and active transcription, the functionality of these marks at promoters remains elusive and no causal correlation has been established (Haberle and Stark 2018). In addition, active promoters have a well define nucleosome organization in which core promoters appear as nucleosome free regions (Mavrich et al. 2008; Valouev et al. 2011), however, an alternative explanation for this observation is that active promoters are enriched on unstable histone variants, as it has been determined in human cells (Jin et al. 2009).\\

In contrast to core promoters, proximal promoters are not mainly bound by the basal transcription factors, but rather by specific transcription factors that enable the binding of coactivators. In consequence, proximal promoters are often considered as proximal enhancers as their features more closely resemble those of enhancers, described below.\\

		\subsection{Enhancers}

Enhancers are \textit{cis}-regulatory elements that boost the expression of their target genes irrespectively of their location relative to promoters, being able to act at long distances from their target promoters. This definition was derived from the seminal discoveries reached by transfecting the rabbit $\beta$-globin gene into HeLa cells using vectors carrying simian virus 40 (SV40) DNA, of which a 72 bp region become the first enhancer identified (Banerji et al. 1981; for a historical review of the discovery of enhancers see Schaffner 2015). Soon after the identification of viral enhancers, cellular enhancers were also characterized denoting another of their characteristics, tissue specificity (Banerji et al. 1983). Enhancers have been identified in a large number of organisms and they outnumber protein coding genes (Creyghton et al. 2010; Aday et al. 2011; Shen et al. 2012; Villar et al. 2015; Visel et al. 2009). In eukaryotes with compact genomes, such as yeast, most of the transcriptional regulation occurs at promoters, nevertheless, gene expression can also be influenced by additional features encoded in the sequence, including 3’ untranslated regions (UTR) and codon biases (Schikora-Tamarit et al. 2018). Importantly, by analyzing the transcriptional regulation of one of the closest unicellular organisms to metazoan it was determined that enhancers with distal activity are an innovation related to multicellularity (Sebé-Pedrós et al. 2016).\\

Enhancers activate their target genes by recruiting transcription factors (TFs) that bind to short DNA motifs, transcription factor binding sites (TFBSs). In turn, TFs interact with co-activators, including Mediator, which participate in the remodeling of chromatin to facilitate transcription initiation (Roeder 2005). The content and distribution of TFBSs within enhancers is commonly known as the enhancer grammar and the first models to describe the association of TFs within enhancers are based on it. The enhanceosome model describes enhancers in which TFs act by direct cooperativity constraining sequence variation, as they have to recognize their TFBSs in a strict order. On the other hand, the billboard model describes enhancers in which cooperativity of TFs is indirect and therefore, they can buffer variations in their TFBSs. While these two independent models are supported by experimental data, the current vision of TF interactions within enhancers points towards a unifying model, in which cooperativity of TFs can be direct or indirect in different regions of the same enhancer and thus, sequence constraints are not uniform along the enhancer (Long et al. 2016). It is generally accepted that TF cooperativity facilitates chromatin remodeling of enhancers to achieve nucleosome eviction leaving TFBSs exposed. An alternative mechanism that has been proposed to guide the recognition of TFBSs within enhancers, relies on the functions of ``pioneer'' TFs. These TFs can prime enhancers to be later bound by additional TFs and become active. Nevertheless, only few pioneer TFs have been identified and the mechanism enabling subsequent binding of TFs remains unclear (Calo and Wysocka 2013; Levine et al. 2014). As suggested for promoters, the presence of unstable histone variants could also mediate the accessibility of TFBSs in enhancers (Calo and Wysocka 2013).\\

Conserved noncoding elements in the genome are enriched on elements showing enhancer functionality. Indeed, the first efforts to systematically identify enhancers genome-wide were guided by sequence conservation and they demonstrated that highly conserved noncoding elements can be enhancers mainly associated with developmental genes (Nobrega et al. 2003; Bejerano et al. 2004; Woolfe et al. 2004; Pennacchio et al. 2006). This type of approach has even been applied to identify conserved enhancers between species in the two distant animal clades of deuterostomes and protostomes (Clarke et al. 2012). However, most enhancers show low conservation at the sequence level (Villar et al. 2015) and growing evidence suggests that among enhancers, active enhancers during development tend to have higher sequence conservation. For example, comparison of evolutionary conservation of active enhancers identified in three tissues during mouse development determined that the most conserved enhancers showing the highest sequence constraints are those active during early embryogenesis (Nord et al. 2013). Also, demethylated regions overlapping active enhancer marks during the phylotypic stage (i.e. they stage at which organogenesis starts and animals have the highest resemblance to other species) in zebrafish and mouse have higher sequence conservation that equivalent regions identified at earlier time points (blastula or gastrula stages) or in adult tissues, suggesting constraints in the enhancers that contribute to the establishment of the body plan in vertebrates (Bogdanović et al. 2016). Analyses of enhancers at specific loci have illustrated that although overall sequence conservation of an enhancer could be low, it is possible to identify conserved small regions/TFBSs flexibly arranged within the enhancer that can confer functional conservation (Hare et al. 2008; Rastegar et al. 2008). This observation has been challenged at the genome-wide level by comparing shared enhancers between \textit{Drosophila melanogaster} and other four \textit{Drosophila} species, concluding that significant differences in the sequence conservation of shared and non-shared enhancers between species are only detectable when analyzing specific TFBSs instead of whole enhancer regions (Arnold et al. 2014). In addition, comparative analyses of sequence conservation of enhancers and promoters in 20 mammalian genomes have determined that enhancers have more rapid evolution than promoters and suggest that new enhancers are preferentially born by exaptation (i.e. a new adaptation relative to the original function) of ancient DNA sequences rather than by expansion of repeats (Villar et al. 2015). As such, notwithstanding the well documented examples of enhancers with high sequence conservation, it is possible to conclude that most enhancers have low sequence conservation, generally restricted to small sequences that most likely correspond to TFBSs.\\ 


		\begin{figure}[h!]
			\centering
			\includegraphics[width=12cm,height=5cm]{figures/Intro_Figure1.pdf}
  			\caption[intro1]{Representation of a DNA loop formed by the interaction of a promoter and its specific enhancer. Adapted by permission from RightsLink: Springer Nature from Shlyueva, et al. 2014.}
			\label{intro1}
		\end{figure}


After the discovery of enhancers and in view of the fact that they can exert their functions distantly from their target genes, it was proposed that enhancer-promoter communication is established via DNA looping. This idea was derived from observations of the lac operator functions in \textit{Escherichia coli} (Oehler et al. 1990). However, it had not been possible to test this hypothesis until the development of two techniques: chromatin conformation capture (3C) and DNA fluorescence in situ hybridization (FISH). 3C works by crosslinking chromatin to obtain a snapshot of the interactions at a given time, followed by digestion, religation and amplification of specific pairs of loci of interest to calculate the frequency of interactions between those genomic regions (Dekker et al. 2002). In contrast, FISH relies on fluorescent probes to bind chromatin loci, thus enabling to measure distances between loci and test for colocalization using microscopy. Importantly, high-throughput versions of the 3C technique started being applied to identify long-range interactions in human cells as part of the ENCODE project (Dunham et al. 2012). The original identification of long-range interactions between 1\% of the human genome showed that enhancer-promoter interactions were enriched among the interactions identified, and together with promoter-promoter interactions, they have the highest cell type specificity (Sanyal et al. 2012). For instance, even genes that have common expression in different cell types can interact with a completely distinct set of enhancers according to the cell type (Kieffer-Kwon et al. 2013). As mentioned above, Mediator has been proposed to act as a bridge between enhancers and promoters and, therefore, to participate in the stabilization of DNA loops. In addition, cohesin is another protein complex known to be involved in DNA looping, see Figure 1 (Kagey et al. 2010). However, enhancer-promoter communication is not necessary a one-to-one mechanism (Sanyal et al. 2012). This can be exemplified by the determination that transcriptional bursting of two promoters controlled by a single enhancer can occur concomitantly in \textit{Drosophila} embryos (Fukaya et al. 2016).\\


Besides their general low sequence conservation, an emerging hallmark of enhancers is active transcription from these loci. The transcripts generated from enhancers are referred as enhancer RNAs (eRNAs), they are transcribed bidirectionally by Pol II, but in contrast to mRNAs, they are not polyadenylated, rarely spliced and exosome-sensitive (Rothschild and Basu 2017). Interestingly, transcription of eRNAs is highly correlated with enhancer activity (Kim et al. 2010; Andersson et al. 2014). Despite this correlation, their functions remain largely elusive and one plausible explanation for their transcription is that they are byproducts of active chromatin regions. However, there are indications that eRNAs could in fact participate in enhancer-mediated regulation. For example, targeted reduction in the levels of an eRNA transcribed from one of the enhancers controlling the expression of \textit{Nanog} and \textit{Dppa3} in mouse embryonic stem cells (mESCs) leads to a decrease in the interaction frequency of the enhancer and \textit{Dppa3} and in the expression of this gene, suggesting that this eRNA could function as a loop stabilizer (Blinka et al. 2016). Future investigation should shed light into the impact of these molecules on enhancer activity and analyze if eRNAs could exert their functions in \textit{trans}, contrarily to what could be inferred from their short lifetimes.\\

		\subsubsection{Identification of enhancers}

In spite of the low sequence conservation of enhancers, it is possible to identify a fraction of enhancers solely by the sequence. Indeed, there are algorithms that take advantage of short regions of sequence conservation to predict functionally conserved regulators within noncoding sequences based on pair-wise comparisons of distant vertebrate genomes (Taher et al. 2011). Nevertheless, most of the analyses aiming to identify enhancers genome-wide are currently based on high-throughput sequencing approaches.\\

Given that enhancers are enriched on TFBSs, one strategy to identify them is to map the binding regions of TFs by ChIP-seq (Whyte et al. 2013; Siersbæk et al. 2014). The disadvantages of this strategy are that since TFs act cooperatively within enhancers, several ChIP-seq assays are required to identify regions of TFBS enrichment and previous knowledge about the TFs binding to enhancers in a particular tissue or cell type is needed. For these reasons, the identification of chromatin marks that label enhancers (Barski et al. 2007; Heintzman et al. 2009) was fundamental to facilitate their annotation. Hence, it is possible to systematically annotate enhancers by performing ChIP-seq with specific antibodies against certain chromatin marks (Nord et al. 2013; Vermunt et al. 2014; Bogdanović et al. 2012) or against the chromatin modifiers that deposit them (Visel et al. 2009). The most frequently used chromatin marks to identify enhancers are H3K27ac that besides labelling active promoters it also labels active enhancers, and mono-methylation of lysine 4 on histone H3 (H3K4me1) that labels poised enhancers (Creyghton et al. 2010; Rada-Iglesias et al. 2011).\\

Alternative methods to identify enhancers include, but are not restricted to, assay for transposase accessible chromatin with high throughput sequencing (ATAC-seq), mapping of differentially methylated regions, and reported based methods, such as, self-transcribing active regulatory region sequencing (STARR-seq) (Arnold et al. 2013; Lee et al. 2015; Quillien et al. 2017). Although the last method concomitantly identifies functional enhancers, it had been difficult to implement it in organisms with bigger genomes than the one of \textit{Drosophila} until later adaptations to analyze specific regions (Vanhille et al. 2015). In addition, STARR-seq was recently coupled with ChIP-seq to add the spatiotemporal information that could not be retrieved from the original protocol and identify functional enhancers in human ESCs (hESCs) (Barakat et al. 2018).\\

Considering that every approach has some disadvantages, the ideal strategy to identify enhancers in vertebrate genomes is to combine several methods to obtain highly reliable annotations. Nonetheless, approaches based exclusively on profiling of chromatin marks can provide a first insight into the enhancer landscape of a given tissue or cell type for further refinement.\\

		\subsubsection{Differences between promoters and enhancers}


		\begin{figure}[h!]
			\centering
			\includegraphics[width=12cm,height=7cm]{figures/Intro_Figure2.pdf}
  			\caption[intro2]{Similarities and differences between promoters and enhancers. (A) Promoters and enhancers share the same architecture of transcriptional initiation. (B) The stability of the RNAs transcribed from promoters and enhancers is different and only promoters are engaged in productive transcription. Adapted by permission from RightsLink: Springer Nature from Weingarten-Gabbay and Segal, 2014.}
			\label{intro2}
		\end{figure}


Approximately 2-3\% of promoters in mouse and human cells are able to enhance expression of distal genes (Dao et al. 2017). Therefore, if promoters can act as enhancers, enhancers can trigger transcription and chromatin marks like H3K27ac are enriched at promoters and enhancers, the vision of two different types of regulators is questioned. It has already been underlined that proximal promoters are usually considered as proximal enhancers of their gene, however, the differences between core promoters and enhancers had remained unclear until few years ago. Identification and comparison of transcriptional initiation sites within enhancers and promoters in human cell lines determined that these sites are largely indistinguishable. The features that were tested to assess similarity were: bidirectional transcription, spacing between the +1 nucleotides, binding of TFs in the central spacer, presence of well-positioned nucleosomes surrounding the TSSs, and Pol II and TFIID enrichment. In spite of having a common architecture at transcription initiation sites, the clear difference between promoters and enhancers is the stability of their transcripts that can be quantified using GRO-seq (global run-on sequencing coupled with enrichment for nascent RNAs with 5’ caps) and CAGE (cap analysis of gene expression) data. Whereas the sense transcript at promoters is highly stable, the upstream antisense transcript (uaRNA) is equally unstable than eRNAs, see Figure 2 (Core et al. 2014). The difference in stability may be related to the presence of splice sites and early polyadenylation signals that stabilize or destabilize transcripts, respectively. It can be then hypothesized that unidirectionality in transcript stability at promoters is an acquired feature (Haberle and Stark 2018). In line with this hypothesis, emergence of novel promoters from exaptation of enhancers has been identified in the primate and rodent lineages, and its emergence is associated with biased increase in GC content and splicing motifs (Carelli et al. 2018).\\


Promoters that have been identified as functional enhancers exhibit particular features in comparison to the rest of promoters. For instance, they are associated with higher levels of H3K4me1, H3K27ac and p300, which is the acetyltransferase that deposits H3K27ac (Dao et al. 2017). Ergo, enhancer activity can be used to identify a sub-classification of promoters rather than being a descriptive feature of promoters.\\

Consequently, a more complete definition of promoters could be: promoters are defined as the sequences surrounding TSSs capable of recruiting basal transcription factors and coactivators to trigger productive transcription of their associated gene or, less frequently, to enhance expression of distal genes.\\


		\subsection{Super-enhancers}

Enhancer analyses in mESCs revealed that important genes in the specification of pluripotency are in close proximity to clusters of enhancers. To systematically identify and characterize these clusters, nearby enhancers that had been identified by enrichment of master TFs (Sox2, Oct4 and Nanog) were stitched and ranked based on Mediator ChIP-seq signal, as it is known that Mediator is enriched at enhancers. The ranking based on Mediator signal showed that most of the signal is accumulated by 231 enhancer clusters, whose constituent enhancers have higher abundance of TF ChIP-seq signal than enhancers outside these clusters. Therefore, the 231 clusters were named super-enhancers to distinguish them from the rest of stitched enhancers, referred as typical enhancers (Figure 3A). Super-enhancers are larger than typical enhancers and are preferentially associated with pluripotency genes in mESCs. Moreover, super-enhancers are highly sensitive to perturbations in the levels of the proteins enriched on their constituent enhancers, as inferred from early and high significant reduction of the expression of super-enhancer associated genes after depletion of Oct4 and Mediator (Figure 3B) (Whyte et al. 2013). Besides having enriched TF and Mediator signal, mESC super-enhancers are also enriched on Pol II, chromatin remodelers and trait-associated single nucleotide polymorphisms (SNPs) (Hnisz et al. 2013). Importantly, it was shown that the best ChIP-seq signal that could more reliable identify super-enhancers when used on its own is the H3K27ac mark (Hnisz et al. 2013).\\

Super-enhancers are not unique gene regulators in mESCs. Indeed, soon after their characterization, super-enhancers were also identified in a large number of cells and tissues including oncogenic cell lines (Lovén et al. 2013; Hnisz et al. 2013). Strikingly, these analyses determined that super-enhancers tend to be associated with cell identity genes (Hnisz et al. 2013). For this reason, and considering the high sensitivity of super-enhancers to perturbations in the levels of their binding factors, it was hypothesized that modifying the levels of these factors could affect the establishment of super-enhancers and trigger changes in cell identity (Lovén et al. 2013). Thus, factors binding to super-enhancers in oncogenic cell lines are potential therapeutic targets (Lovén et al. 2013; Minzel et al. 2018). Importantly, newly developed small molecules inhibiting the casein kinase 1A1 can also target the catalytic subunits of TFIIH and a transcription elongation factor, leading to the collapse of acute myeloid leukemia super-enhancers, which show enrichment of these factors. As such, treatment using these small molecules triggers a synergistic mechanism of p53 activation, following casein kinase 1a1 inhibition, and inactivation of oncogenes associated with super-enhancers that results in the elimination of leukemic stem cells and relieve of disease signs in leukemia mouse models (Minzel et al. 2018).\\


		\begin{figure}[h!]
			\centering
			\includegraphics[width=15cm,height=7cm]{figures/Intro_Figure3.pdf}
  			\caption[intro3]{Characteristics of typical enhancers and super-enhancers. (A) Genome browser screen-shots showing the ChIP-seq signal of pluripotent state TFs around the loci of a typical enhancer and a super-enhancer. (B) Cartoon and curves representing the differences of dependance on activator abundancy and transcriptional activity for typical enhancers and super-enhancers. Adapted by permission from RightsLink: Elsevier from Whyte et al. 2013 and Lov\'en et al. 2013.}
			\label{intro3}
		\end{figure}


Apart from their original described characteristics, posterior analyses of super-enhancers have revealed additional characteristics. For instance, it has been shown that in contrast to constitutive enhancers within typical enhancers, those located within super-enhancers are enriched on small regions ($\sim$250-400 bp) highly bound by several TFs denoted as epicenters or hotspots (Siersbæk et al. 2014; Adam et al. 2015). This characteristic supports the idea that super-enhancers may represent more than simple clusters of enhancers. Analysis of epicenters in hair follicle stem cells \textit{in vivo} and \textit{in vitro} showed the high dynamism of super-enhancers, given that when cells are exposed to a new microenvironment, only a small fraction of them maintains their super-enhancer status. Interestingly, super-enhancers that are constant in different conditions can present shifts in the epicenters that are used within the super-enhancer region (Adam et al. 2015). These results suggest a mechanism by which TFs could be reusing pre-established active chromatin regions to achieve fast modulation of cell identity. Moreover, the only hair follicle stem cell TF gene that maintains its association with a super-enhancer both \textit{in vivo} and \textit{in vitro} codes for the TF SOX9. Strikingly, this TF is able to induce the establishment of hair follicle stem cell super-enhancers when expressed ectopically, indicating that pioneer TFs can also participate in super-enhancer establishment (Adam et al. 2015).\\

As already mentioned, super-enhancers have been involved in oncogenesis. In B cell lymphomas it is frequent to observe translocations of oncogenes promoted by the activation-induced cytidine deaminase (AID). Characterization of off-target AID regions in mouse and human B cells have determined that they accumulate at super-enhancers that overlap exons. These super-enhancers favor antisense transcription that generates regions susceptible to AID activity, explaining why translocated regions in B cell lymphomas commonly include lineage specific genes (Meng et al. 2014). Another mechanism by which super-enhancers promote oncogenesis is by inducing the overexpression of oncogenes. An example of this mechanism occurs in cases of T cell leukemias, in which mutations upstream of the \textit{Tal1} TSS lead to formation of MYB TFBSs. Binding of MYB to these sites induces the formation of a super-enhancer that results in \textit{Tal1} overexpression, demonstrating that formation of super-enhancers can be triggered by a single nucleation site (Mansour et al. 2014). Also, in proliferating lymphoblastoid cell lines generated by Epstein-Barr virus infection, super-enhancers are established by cooperative binding of pathogen and host TFs, increasing the expression of the \textit{MYC} and \textit{BCL2} oncogenes that leads to high proliferation and survival (Zhou et al. 2015).\\

Strikingly, super enhancers can also stimulate the processing of primary microRNA transcripts by recruiting the microprocessor complex through cooperative activity of their enhancer constituents (Suzuki et al. 2017). Given the functions of super-enhancers in regulation of gene expression, genome stability, oncogenesis and RNA processing, it is congruent that they play a central role in regulatory circuitries that respond to signaling pathways and consequently modify cell identity programs (Hnisz et al. 2015; Lin et al. 2016; Saint-André et al. 2016). Nevertheless, characterization of super-enhancer has been mainly restricted to mammalian genomes and the evolution of these regulators had not been investigated.\\

The described characteristics and functions of super-enhancers highlight their importance in the establishment of cell fate. However, to have a comprehensive understanding of their mechanisms of action it is necessary to analyze them considering the actual nuclear context, in which genomes are organized in three-dimensional (3D) space. The next section summarizes the current understanding of the 3D genome organization and its strong relationship with super-enhancers and other \textit{cis}-regulatory elements.\\

	\section{3D genome organization in vertebrates and distant eukaryotes}

As already mentioned, the development of 3C and FISH techniques was essential to enable the identification of interactions between enhancers and promoters. More broadly, these techniques offered the possibility to study the 3D genome organization. 3C-based methods have been the preferred choice to identify genomic regions with high frequency of interaction, and FISH is commonly used as an orthogonal method to support 3C results by testing colocalization of loci.\\

		\subsection{Identification of chromatin interactions}


		\begin{figure}[h!]
			\centering
			\includegraphics[width=16cm,height=22.5cm]{figures/Intro_Figure4.pdf}
  			\caption[intro4]{Schematic description of the most frequently used 3C-based techiniques. Adapted from Denker and de Laat 2016 (Creative Commons License).}
			\label{intro4}
		\end{figure}


All 3C-based techniques rely on a first step to crosslink chromatin that is generally performed using formaldehyde, which forms covalent bonds between interacting chromatin regions at $\sim$2 {\AA} distances (Hoffman et al. 2015). Chromatin is then sheared by restriction enzymes or sonication, and the sheared chromatin is ligated to generate fragments composed of interacting loci (Dekker et al. 2002). These are the common steps that are shared between 3C and all its derived techniques, but each of them has additional steps to interrogate different interactions, as seen on Figure 4 that summarizes how libraries are generated with the more frequently used techniques. In contrast to 3C, all of these techniques use sequencing instead of qPCR to identify interactions in the generated libraries. In circularized 3C (4C), the fragments follow a second round of digestion and ligation, generating small circles that are amplified using primers for a specific region (``the viewpoint''). Therefore, 4C gives information about the interactions of one region and the rest of the genome. A closely related method to 4C is Capture-C, in which the fragments are also re-digested but they are then directly amplify using sequencing primers. The amplified fragments are then captured using specific biotinylated oligonucleotides for one or several viewpoints. For 3C carbon copy (5C) libraries, the ligated fragments are hybridized with primers designed to cover one region of interest. The primers contain sequencing adaptors, thus enabling the amplification for sequencing. Hence, 5C can identify all chromatin interactions within a specific region of interest. 3C can also be coupled with immunoprecipitation by pooling the sheared fragments using one antibody and filling the sonicated ends with biotinylated nucleotides that will be used to captured the ligated fragments after decrosslinking (Denker and De Laat 2016). This method is called chromatin immunoprecipitation analysis by paired-end tag sequencing (ChIA-PET) and currently, there is also an improved protocol that follows the same principle (HiChIP) to identify interactions mediated by specific proteins (Mumbach et al. 2016). Finally, the 3C-based techniques that permit the interrogation of all interactions in the genome are Hi-C, Micro-C and DNase Hi-C. In contrast to Hi-C in which chromatin is sheared using restriction enzymes, in Micro-C and DNase Hi-C, chromatin is sheared using MNase and DNase, respectively. For these techniques, ligation is performed using biotinylated nucleotides, enabling the specific recovery of ligated fragments after one round of sonication. In addition, there are modified versions of Hi-C that introduce one extra step to capture regions of interest as for Capture-C (Denker and De Laat 2016). For the ease of implementation and the genome-wide data generated by Hi-C, this method has been largely applied to study genome organization in multiple organisms (Lazar---Stefanita et al. 2017; Lieberman-Aiden et al. 2009; Dixon et al. 2012; Vietri Rudan et al. 2015; Sexton et al. 2012). Of note, the improved protocol used to perform Hi-C is referred as \textit{in situ} Hi-C, because crosslinking is carried out in intact nuclei before cell lysis (Rao et al. 2014), which is the current standard followed by other techniques, such as, HiChIP (Mumbach et al. 2016).\\

Analysis and visualization of information generated by the 3C-based techniques here described changes according to the extent of interactions identified. In the case of Hi-C and other methods that focus on the detection of interactions genome-wide, the information is generally represented as symmetric heat maps of chromosomes, referred as contact maps, which are binned in non-overlapping windows of fixed size. The resolution of the contact map is then equal to the size of the window used to generate it. In a contact map, each cell then reflects the interaction frequency between two loci, as such, the cells closer to the diagonal represent the interactions at shortest linear distances. The interaction frequencies are computed using the valid pairs of sequencing reads that are identified by applying strict filters to keep only the pairs originating from distant fragments that were ligated (Servant et al. 2015). In order to obtain normalized interaction frequencies different methods have been developed to process the raw number of valid pairs. One of the more commonly used methods for its ease of computation is the ICE (iterative correction and eigenvector decomposition) method that is implemented under the assumption that all loci in a genome should have equal visibility (Imakaev et al. 2012).\\

		\subsubsection{Orthogonal methods to 3C}

Although 3C-based techniques and FISH aim to identify chromatin interactions, their results are not interchangeable because they provide different measures as a \textit{proxy} to identify interactions. In the case of the 3C-based techniques, their results are averages of cell populations that reflect probabilities in the frequency of interaction that depend on proximity. On the other hand, FISH is used to measure distances in space in single cells. In consequence, it is logical to infer that most of the discrete interactions between loci that are identified by 3C-based techniques are presented only in a subpopulation that deviates from the average that can be calculated from hundreds of distances measured in single cells (Dekker 2016; Fudenberg and Imakaev 2017). Therefore, it is not surprising to observe discrepancies between the results obtained by these methods, indeed, it has been shown with polymer models that dynamism of interactions is one of the potential causes of discrepancy (Fudenberg and Imakaev 2017). While these arguments emphasize the need to design appropriate validation experiments (Giorgetti and Heard 2016), they also urge for additional orthogonal methods that could confirm and expand the conclusions obtained with 3C-based techniques and FISH.\\

Importantly, during the last years, three orthogonal methods to Hi-C and FISH have been developed. First, CLING (CRISPR/Cas9 live cell imaging), as indicated by its name, is an imaging technique capable of identifying interacting loci in living cells by exploiting the CRISPR (clustered regularly interspaced short palindromic repeats) system and a non-catalytic CRISPR-associated protein 9 (dCas9). To visualize regions, three single guide RNAs (sgRNAs) containing MS2 or PP7 repeats are designed to target each of the regions of interest. Subsequently, tagged fluorescent proteins modified to recognize the repeats present in the sgRNAs are recruited to the loci bound by the sgRNA-dCas9 complexes enabling their detection. CLING enables the identification of punctuate inter-chromosomal interactions that are not easily identified by Hi-C, this might be a consequence of inter-chromosomal contacts occurring at longer distances than those permitting fixation by crosslinking (Maass et al. 2018b). The second method, split–pool recognition of interactions by tag extension (SPRITE), has the advantage of overcoming the limitations imposed by proximity ligation. The first steps also include crosslinking and shearing of chromatin like in Hi-C, however, instead of ligating the complexes, they are split in 96-well plates and ligated to unique tags for each well. The complexes are then pooled and the process is repeated iteratively to increase the probability of generating unique barcodes for each complex. These barcodes are used after sequencing to cluster the reads corresponding to DNA and RNA molecules from each complex. By applying SPRITE in mESCs, two hubs of interacting regions have been identified involving active or inactive chromatin regions exclusively. Interestingly, the active hub localizes to nuclear speckles, while the inactive hub localizes to the nucleolus, these type of interactions with nuclear bodies have not been identified by only applying Hi-C (Quinodoz et al. 2018). The third method, genome architecture mapping (GAM), is based on cryosectioning of cells followed by laser dissection of nuclei to obtain fine sections from which DNA is extracted. Extracted DNA from each section is then used to build libraries that are sequenced by whole genome amplification. Reads from each library represent the regions that were in proximity in a given nucleus, hence, by combining the co-segregation of pair loci (i.e. presence or absence of specific pairs) of all libraries, it is possible to infer preferred interactions. GAM performed using $\sim$400 sections of mESC nuclei showed that interactions between enhancers and active genes are one of the most frequently identified categories of interactions. Importantly, given that all DNA for each section is used to build the libraries, GAM also enables the identification of more complex interactions, such as triplets. Indeed, it was determined that interactions between structural domains containing super-enhancers are enriched among the triplets identified, highlighting the role of super-enhancers in the organization of the 3D genome (Beagrie et al. 2017). For the moment, these methods have already proved to be useful for the analysis of the 3D genome organization and in the next years they will be instrumental to understand this organization. Importantly, SPRITE and GAM have confirmed the main principles of genome organization in terms of the domain structures that were originally characterized by 5C and Hi-C (Lieberman-Aiden et al. 2009; Dixon et al. 2012; Nora et al. 2012).\\

		\subsection{Principles of 3D genome organization}

The chromatin fiber is not stochastically distributed within the cell nucleus and during interphase each chromosome is preferentially segregated in defined volumes called chromosome territories, which can overlap with their neighboring territories and cause chromatin intermingling (Maass et al. 2018a). In line with previous low-throughput methods, the inter-chromosomal distances calculated with the first low-resolution human Hi-C data confirmed the distribution of chromosomes in territories and the fact that small chromosomes interact more frequently with each other. The latter could be explained by the colocalization of these chromosomes at the center of the nucleus (Lieberman-Aiden et al. 2009). Preferential interactions between chromosomes according to their sizes have also been observed in mouse fibroblast and sperm cells (Battulin et al. 2015). Importantly, by calculating the Pearson correlation on contact maps, it is possible to observe a sharp plaid pattern, indicating compartmentalization of the genome in two types of domains (A and B) dependent on the chromatin state (Figure 5A). Loci localized into A compartment domains show high positive correlation with gene density, gene expression and accessibility, indicating that these loci are active chromatin regions. In contrast, loci corresponding to B compartment domains show anti-correlation with chromatin accessibility and decay of their interaction frequencies occurs at longer distances, suggesting that these loci have higher compaction and are inactive chromatin regions (Lieberman-Aiden et al. 2009). A and B compartment domains have also been associated with early and late replicating regions in the genome (Dixon et al. 2012). Furthermore, analyses of high resolution Hi-C data in human cells have determined that, according to enrichment of different chromatin marks and replication timing, A and B compartment domains can be further subdivided in at least six classifications in total (Rao et al. 2014).\\

Inspection of Hi-C and 5C contact maps of \textit{Drosophila} embryos and mouse and human cell lines at resolutions $\sim$100 kb shows the presence of squares along the diagonal (or triangles when only the upper/lower triangular of the contact map is displayed), which are indicative of the existence of discrete regions of self-interacting chromatin in bulk cell populations (Sexton et al. 2012; Dixon et al. 2012; Nora et al. 2012). These regions are referred as topologically associating domains or contact domains, examples are depicted in Figure 5B. Interestingly, comparison of contact domains and polytene bands in \textit{Drosophila} have determined that although there is not an exact correspondence between these two features, contact domains coincide with polytene bands, whereas inter-bands coincide with chromatin regions between contact domains (Ulianov et al. 2016). In mESCs, it has been calculated that $\sim$91\% of the genome is covered by contact domains identified by Hi-C and importantly, the boundaries of these domains showed enrichment on certain genomic features. For instance, housekeeping genes, transfer RNA (tRNA) genes and chromatin marks associated with active transcription, including H3K4me3 and tri-methylation of lysine 36 on histone H3 (H3K36me3) are among the features enriched at boundaries (Dixon et al. 2012). Notably, it was shown that binding sites of the CCCTC-binding factor (CTCF) are also enriched at contact domain boundaries indicating a role of its insulating activity in the formation of boundaries. However, since CTCF sites are also present at other genomic locations, additional factors or characteristics are necessary to form a boundary (Dixon et al. 2012; Nora et al. 2012). Expression analysis also determined that genes located within the same contact domain have correlated expression indicating that these regions establish confined volumes that favor interactions of genes with their regulatory elements (Nora et al. 2012; Symmons et al. 2016). Indeed, this was first confirmed by showing that the deletion of a boundary leads to the formation of ectopic interactions between domains (Nora et al. 2012).\\

Contact domains are relatively stable between different cell types of the same organism (Dixon et al. 2012, 2015; Rao et al. 2014; Ulianov et al. 2016) and even between different species (Dixon et al. 2012; Vietri Rudan et al. 2015). In line with the observed conservation of contact domains between species, it has been recently determined that rearrangement breakpoints in vertebrate genomes relative to the human genome are enriched at boundaries, but depleted within contact domains. Therefore, it can be hypothesized that there are selective pressures that favor the conservation of contact domains as whole genomic blocks, which may be selectively maintained to preserve interactions between genes and their regulatory elements (Krefting et al. 2018). This hypothesis is also supported by the fact that transposons are depleted within contact domains in the human genome that coincide with large blocks of conserved noncoding elements (Harmston et al. 2017). Conservation of contact domains is congruent with previous results showing that gene misregulation can be caused by exposure to non-canonical regulators, as consequence of modifications in contact domains that possibly lead to detrimental effects for an organism (Ibn-Salem et al. 2014; Lupiáñez et al. 2015; Franke et al. 2016; Symmons et al. 2016). For example, in humans it has been characterized that duplications around the \textit{SOX9} loci can generate aberrant phenotypes, such as, sex reversal and Cooks syndrome, but these effects are not observed in all humans with duplications. By using 4C, it was determined that only duplications expanding contact domain boundaries can establish new contact domains in duplicated regions. To characterize the consequences of new contact domains, mouse models were generated with equivalent duplications that those present in human patients. Analysis of capture Hi-C data showed that when duplications expand both the \textit{Kncj2} loci (closest upstream gene of \textit{Sox9}) and \textit{Sox9} enhancers, mice gain a contact domain and recapitulate defects observed in Cooks syndrome patients. The defects are due to the formation of ectopic contacts between \textit{Kncj2} and the enhancers, leading to changes in the expression pattern of this gene. In contrast, when duplications expand only the \textit{Sox9} enhancers, no changes in expression are detected, as enhancers stay confined in their newly established contact domain (Franke et al. 2016).\\


		\begin{figure}[h!]
			\centering
			\includegraphics[width=14cm,height=19cm]{figures/Intro_Figure5.pdf}
  			\caption[intro5]{Architectural features observed during interphase at different resolutions on a contact map. (A) At low resolutions the inter-chromosomal interactions indicate the distribution of the genome in chromosome territories, whereas the intra-chromosomal maps show a plaid pattern corresponding to the interactions between active and inactive regions (compartment domains). (B) Discrete regions of self-interacting chromatin (contact domains) are observed by zooming in a compartment domain. (C) Contact domains can be anchored by chromatin loops but loop domains are also present within contact domains. Loop domains are visualized on the map by punctuate enrichment of interactions at the corners of domains. Adapted from Razin and Ulianov 2017 (Creative Commons License).}
			\label{intro5}
		\end{figure}


Whereas the overall landscape of contact domains between different cell types is shared, other levels of the genome organization are more dynamic and correlate with the establishment of different transcriptional programs. It has been shown that during differentiation of hESCs, $\sim$40\% of the genome switches from compartment domain (A to B, B to A), implying that although the genomic regions of contact domains are stable, they can uniformly modify their chromatin state that translates to shifts in compartment domain association (Dixon et al. 2015). Contact domains shifting from compartment domains have also been identified during senescence of human fibroblasts (Criscione et al. 2016) and by comparison of 21 human cell types and tissues (Schmitt et al. 2016). In addition, the intra-domain interactions of contact domains can also be rewired, demonstrating the dynamism within contact domains between different cell types (Dixon et al. 2015, 2012; Bonev et al. 2017). In particular, DNA loops between enhancers and promoters account to a fraction of the rewired interactions that occur within contact domains (Ji et al. 2016; Bonev et al. 2017) and cell-type specific loop domains correlate with regions showing differential gene expression between cell types (Rao et al. 2014). In a contact map, loop domains can be visualized as discrete peaks of high interactions between loci that show lower interactions with the intervening adjacent loci (Figure 5C). However, not all loop domains are cell-type specific, as comparison of loop domains identified in eight human cell types showed that these domains can also be conserved, including loops between enhancers and promoters (Rao et al. 2014). Altogether, these results indicate that punctuate chromatin interactions correlate with differences in transcriptional programs that do not have a global impact in the distribution of contact domains, but they are reflected in the chromatin state of domains.\\

		\subsection{Establishment of structural domains}

Given that contact domains appear as stable units in genomes, it has been of high interest to investigate how and when they become established. One of the processes in which the dynamics of contact domains have been studied is embryonic development. Early development is marked by the maternal to zygotic transition during which the maternal transcriptome is degraded and the zygotic genome becomes activated. Synchronous mice and \textit{Drosophila} embryos before and after this transition have been used to analyze the dynamics of contact domains by Hi-C. These studies have shown that contact domains are not clearly defined before the genome activation (Hug et al. 2017; Ke et al. 2017), and although some contact domains are present in the minor activation of the genome in \textit{Drosophila} embryos, the number of contact domains and their insulation values increase during development (Hug et al. 2017). In addition, chemical inhibition of transcription do not interfere with the establishment of contact domains, but there is an increase in the inter-interactions of contact domains and a decrease in the intra-interactions (Hug et al. 2017). Contrary to transcriptional activation, DNA replication has been shown to be necessary to establish contact domains (Ke et al. 2017; Nora et al. 2017).\\

In spite of its apparent minor impact in the establishment of contact domains during the maternal to zygotic transition, transcription does exert an impact on the 3D genome organization. For instance, during differentiation of mESCs to neural progenitor cells, the regions that have increase transcriptional activity also become more structurally complex (Zhan et al. 2017). This relationship between transcription and structural complexity is in line with a previous study showing that regions containing escapee genes in the inactive X chromosome in mouse cells maintain their organization in contact domains (Giorgetti et al. 2016). Nevertheless, a causal relationship between transcription and genome organization could not be directly established as it is unclear if transcription is guiding the organization within those regions, or if the chromatin is organized in a permissive conformation that enables transcription. Strikingly, by analyzing post-mitotic human macrophages infected with influenza A virus a direct effect of transcriptional elongation in the 3D genome organization has been recently determined (Heinz et al. 2018). In these cells, a viral protein guides transcriptional elongation by Pol II and read-through of transcription termination sites of highly induced genes after infection. Read-through transcription leads to decompaction of these loci and shifts from B compartment domain to A compartment domain. Decompaction is explained by disruption of DNA loops mediated by cohesin and anchored at CTCF sites, as chemical inhibition of elongation causes cohesin accumulation and strengths the interactions between loop anchors. Also, analysis of cohesin binding after elongation inhibition suggests that cohesin is evicted from chromatin during transcription rather than displaced concomitantly with transcription. Importantly, the effect of transcriptional elongation in the 3D genome organization is also observed in non-viral stimulated cells and during steady-state transcription (Heinz et al. 2018). Interestingly, although transcription can influence the stability of DNA loops, induction of transcription is not enough to increase insulation in specific loci to form contact domain boundaries in mESCs (Bonev et al. 2017).\\

Of note, contact domains are only detectable during interphase. Analyses of the genome organization at different stages of the cell cycle in synchronous human cells determined that compartment and contact domains are lost during mitosis (Naumova et al. 2013). Also, the decay of the frequency of interactions by linear distance is not in agreement with the expected from a fractal globule polymer, as observed during interphase (Lieberman-Aiden et al. 2009), but rather with the decay expected from an equilibrium globule polymer (Naumova et al. 2013). Major reconfiguration of the genome during the cell cycle has also been observed in the unicellular eukaryote \textit{Saccharomyces cerevisiae} (Lazar---Stefanita et al. 2017). For these reasons, the cell cycle is considered as the major determinant of the 3D genome organization (Nagano et al. 2017). Indeed, by analyzing single cell Hi-C data of mouse cells it was shown that contact domains are more insulated during the G1 phase, whereas the compartmentalization of the genome is more evident at G2 phase, suggesting that the mechanisms that establish contact and compartment domains are uncoupled (Nagano et al. 2017). Importantly, single cell Hi-C data of mice oocytes have led to the conclusion that contact domains are not fixed blocks present in all cells, but they rather represent the average regions of preferential interactions occurring in individual cells (Flyamer et al. 2017). This can explain why deletions at contact domain boundaries do not necessary cause collapsing of adjacent domains (Franke et al. 2016).\\


		\begin{figure}[h!]
			\centering
			\includegraphics[width=16cm,height=10cm]{figures/Intro_Figure6.pdf}
  			\caption[intro6]{Scheme depicting how the linear orientation of CTCF sites impacts the formation of loop domains and inner-loops (shown on toop). Green boxes represent the 11 zinc fingers of CTCF. Cohesin complexes are represented as orange rings. CBS, CTCF binding site. Adapted by permission from RightsLink: Elsevier from Guo et al. 2015.}
			\label{intro6}
		\end{figure}


CTCF and cohesin are enriched at contact domain boundaries (Ji et al. 2016; Dixon et al. 2012; Van Bortle et al. 2012; Dowen et al. 2014). Indeed, CTCF ChIA-PET loops recapitulate contact maps obtained by high resolution Hi-C of human cells, contrary to loops identified by Pol II ChIA-PET, which are smaller than CTCF loops and generally contained within them (Tang et al. 2015). Strikingly, it was determined that anchors of contact domains and loop domains are mainly observed between loci containing CTCF sites in convergent orientation; therefore, there is enrichment of divergent CTCF sites at boundaries of contact domains (Rao et al. 2014; Guo et al. 2015; Vietri Rudan et al. 2015; de Wit et al. 2015; Tang et al. 2015). In contrast, CTCF sites are generally found in tandem orientation within contact domains, see Figure 6 (Tang et al. 2015). Also, it has been shown that deletions of CTCF sites at boundaries enable the formation of ectopic interactions between enhancers and promoters of adjacent contact domains (Nora et al. 2012; Lupiáñez et al. 2015; Hnisz et al. 2016; Ji et al. 2016; Dowen et al. 2014). Genomic inversions of CTCF sites of converging pairs, which are less invasive genetic modifications and preserve binding of CTCF and cohesin, have also confirmed that the orientation of CTCF sites is relevant in the establishment of chromatin interactions (de Wit et al. 2015; Guo et al. 2015). For these reasons, CTCF is considered as a major regulator of the 3D genome organization in mammals.\\

Although CTCF is conserved in Metazoa (Heger et al. 2012), its central role in genome organization determined in mammals cannot be extrapolated to all organisms belonging to the Metazoa clade. For instance, contact domains annotated by high resolution Hi-C data from \textit{Drosophila} cells have indicated that although architectural proteins are enriched at their boundaries, CTCF binding is only present in the vicinity of $\sim$28\% of them and CTCF sites do not have a preferred orientation. In contrast, contact domain boundaries coincide with loci with high levels of transcriptional activity, assessed by GRO-seq, independently of the abundance of architectural proteins. Strikingly, modelling of Hi-C maps exclusively using GRO-seq data shows high correlation with the actual Hi-C maps. Even though transcription is considered a major determinant of genome organization in \textit{Drosophila}, architectural proteins in general can also be involved in its organization, although to a lesser extent, as loci separated by more sites bound by these proteins have lower interaction frequencies (Rowley et al. 2017). These results are in agreement with previous modelling simulations suggesting that genome organization in \textit{Drosophila} is based on activity rather than on architectural proteins (Ulianov et al. 2016). Major dependence on transcription was also suggested for other non-mammalian eukaryotes including \textit{Caenorhabditis elegants} and \textit{Arabidopsis thaliana} by modelling of contact maps with RNA sequencing (RNA-seq) data (Rowley et al. 2017). Contrary, human GRO-seq data is not enough to recapitulate Hi-C contact maps and the domains that are observed by modelling with GRO-seq data correspond to compartment domains, while models based on CTCF ChIP-seq data only represent the distribution of contact domains. In consequence, only the maps modelled by GRO-seq data combined with CTCF ChIP-seq data have high concordance with actual contact maps in human (Rowley et al. 2017). Besides indicating that CTCF roles in human and other eukaryotes are different, these results also highlight that two independent modes can act to organized the genome in structural domains.\\

Despite the growing evidence indicating that deletions of CTCF sites at contact domain boundaries can enable the formation of ectopic enhancer-promoter interactions, the impact of CTCF and cohesin depletion in the genome organization remained unexplored due to their essentiality in mammals (Fedoriw et al. 2004). However, inducible depletions of CTCF and the loader and one subunit of cohesin were recently achieved by conditional depletion using the degron system in mouse cells. These studies determined that when CTCF or cohesin related proteins are degraded, contact and loop domains are vanished as the insulation of domains decreases. Furthermore, this process is accompanied by an increase of genome compartmentalization (Rao et al. 2017; Nora et al. 2017; Schwarzer et al. 2017). Congruent with this, it was previously shown that although chromatin marks labelling inactive regions align with contact domains, the lack of such marks does not affect the establishment of contact domains in the X-inactivation center in mouse cells (Nora et al. 2012). In consequence, these results support the notion that the 3D genome organization in mammals is regulated by two modes, one that is dependent of CTCF and cohesin and participates in the formation of contact and loop domains and another that occurs independently of CTCF and cohesin to establish active and inactive chromatin domains. The models explaining the two modes of genome organization that have been proposed, tested and supported by simulations and experimental data are described below.\\

		\subsection{Phase separation model}

The model that has been suggested to explain the establishment of compartment domains is based on analyses of super-enhancers interactions and the substantial differences observed between A and B compartment domains relative to histone modifications (Lieberman-Aiden et al. 2009; Rao et al. 2014; Whyte et al. 2013; Lovén et al. 2013). In general this model, called phase separation, establishes that chemical reactions occurring during the interaction of protein and nucleic acids can lead to the formation of isolated multi-molecular assemblies resembling membraneless organelles (Hnisz et al. 2017). Phase separated states in cells have already been determined, for example, the heterochromatin protein 1 (HP1) that forms liquid-droplets \textit{in vitro} can be assembled in foci that follow the dynamics of phase separated compartments in \textit{Drosophila} embryos and mouse cells, indicating that phase separation mediates the establishment of heterochromatin (Strom et al. 2017). Also, characterization of DNA isolated from nuclear ribonucleoprotein complexes in mESCs has shown that these DNA sequences correspond to the loci of highly expressed genes and super-enhancers, which supports the formation of assemblies to facilitate gene regulation (Baudement et al. 2018).\\

In mESCs, $\sim$84\% of the super-enhancers are organized in insulated neighborhoods demarked by cohesin binding (Dowen et al. 2014). Furthermore, super-enhancers participate in complex interactions with multiple target genes simultaneously (Novo et al. 2018), show spatial colocalization (Beagrie et al. 2017) and selective depletion of one of the cohesin subunits leads to the formation of large cliques of super-enhancer loci (Rao et al. 2017). Similarly to the compartmentalization mediated by HP1, it has been shown that two proteins enriched on mESC super-enhancers, Mediator subunit 1 (MED1) and BRD4, also form foci with liquid-droplets dynamics (Cho et al. 2018b; Sabari et al. 2018), and these foci can colocalize with super-enhancers (Sabari et al. 2018). In addition, \textit{in vitro} assays in human cells showed that MED1 foci also compartmentalize BRD4 and Pol II (Sabari et al. 2018).\\


		\begin{figure}[h!]
			\centering
			\includegraphics[width=15cm,height=20cm]{figures/Intro_Figure7.pdf}
  			\caption[intro7]{Phase separation model applied to super-enhancer networks. (A) Representation of macromolecules involved in enhancer-promoter interactions. (B) Minimal components of the phase separation model. (C) Dynamics of transcriptional activity using a simple model to represent super-enhancers and typical enhancers at different valency values and a fixed equilibrium constant. The inset corresponds to a logarithmic curve showing the dependence of the Hill-coefficient on the number of chains. N, number of chains.  Adapted by permission from RightsLink: Elsevier from Hnisz et al. 2017.}
			\label{intro7}
		\end{figure}


By focusing on super-enhancers it is possible to illustrate how phase separation can mediate genomic compartmentalization (Hnisz et al. 2017). When enhancer-promoter communication is established, the molecules found at these loci (i.e. DNA, nucleosomes, eRNAs, Pol II, the splicing machinery…) have the potential to form interactions (Figure 7A), hence, this can be extrapolated to all the constituent enhancers of a super-enhancer and their target genes. In a simple model, each molecule can be define as a ``chain’’, where each chain can be reversible modified at specific residues to enable interactions between chains. The number of modified residues defines the valency and its dynamics vary according to an equilibrium constant describing the presence or absence of interactions, the phase separated state is reached when chains engage in a critical number of interactions (Figure 7B). Super-enhancers can be then considered as assemblies of a higher number of chains (N) than typical enhancers. Thus, by performing simulations using two different number of chains representing super-enhancers and typical enhancers, and fixed equilibrium constant and valency values it is possible to observe that super-enhancers reach maximum transcriptional activity (i.e. phase separated state, as the size of the largest chain cluster equals the total number of chains) faster than typical enhancers. In addition, cooperative binding of super-enhancer complexes should then be higher than the one of typical enhancer clusters, as reflected by the Hill coefficient that is directly dependent on the number of chains (Figure 7C). An interesting observation from this simulation is that there is a critical valency value at which super-enhancers cluster more rapidly until reaching saturation, whereas transcriptional activity of typical enhancers increases more smoothly (Hnisz et al. 2017). Therefore, the existence of this critical value explains why super-enhancers are highly sensible to perturbations in the concentration of potential modifiers of the valency, such as BRD4 (Lovén et al. 2013; Hnisz et al. 2017).\\

Besides providing a framework that can explain the concomitant regulation of multiple target genes by an enhancer, phase separation also explains how cell fate can be established without the need of special molecules, but just by adjusting the concentrations of common regulators (Hnisz et al. 2017).\\

		\subsection{Loop extrusion model}


		\begin{figure}[h!]
			\centering
			\includegraphics[width=16cm,height=9cm]{figures/Intro_Figure8.pdf}
  			\caption[intro8]{Loop extrusion model. (A) Components of the loop extrusion model and different events that can take place during time. (B) Chromatin dynamics under the loop extrusion model. Adapted from Fudenberg et al. 2016 (Creative Commons License).}
			\label{intro8}
		\end{figure}


Loop extrusion is a model by which loop-extruding factors and boundary elements control the organization of polymer fibers. Once the loop-extruding factors are loaded in a polymer, they slide on it until they encounter boundary elements, see Figure 8 (Sanborn et al. 2015; Fudenberg et al. 2016). Based on the distribution of cohesin subunits and CTCF binding sites in the genome (Rao et al. 2014), it has been proposed that they act as the extruders and boundary elements, respectively (Sanborn et al. 2015; Fudenberg et al. 2016). Indeed, simulated contact maps generated applying the loop extrusion model with CTCF ChIP-seq data and considering the orientation of CTCF sites show high correlation with Hi-C contact maps, and the formation of contact domains (Sanborn et al. 2015). Also, loop extrusion simulations of chromatin interactions in individual cells predict that contact domains emerge when the signal of preferential interactions in a population of cells is averaged, as confirm using single cell Hi-C data (Fudenberg et al. 2016; Flyamer et al. 2017).\\

Additional evidence that CTCF and cohesin are involved in formation of loops was found by analyses of interacting proteins and regulatory subunits of cohesin. For instance, depletion of WAPL and PDS5 causes stabilization of cohesin on DNA and the formation of longer loops (Haarhuis et al. 2017; Wutz et al. 2017; Gassler et al. 2017), whereas, the SCC2/SCC4 cohesin loader is important for loop processivity (Haarhuis et al. 2017). Furthermore, loop extrusion can also favor the formation of strong regions of interaction visualized on a Hi-C contact map as stripes perpendicular to the diagonal. This type of signal can emerge when one of the extruding subunits is directly bound close to a boundary element and the second subunit has to slide until it finds another boundary element. In mouse B cells these strong interactions have been observed and interestingly, 66\% of the super-enhancers are engaged in this type of interaction, suggesting a mechanism by which super-enhancers could establish interactions with their distant target genes (Vian et al. 2018).\\

		\subsection{Additional regulators of genome organization}

As it has been described above, there are several regulators of the 3D genome organization. Specifically in mammals, loops can be formed by enhancer-promoter communication, super-enhancers establish neighborhoods to regulate their target genes and CTCF interacts with cohesin to strength chromatin interactions and participate in the formation of contact and loop domains. However, it is possible that additional regulators play roles in the genome organization, including the example that has already been mentioned of an eRNA that is involved in the interaction of the \textit{Dppa3} promoter and its enhancer (Blinka et al. 2016).\\

Besides eRNAs, other classes of noncoding RNAs have also been associated with genome organization, such as, long noncoding RNAs (lncRNAs) that are similar to mRNAs but do not have protein coding potential (Marchese et al. 2017). For example, the inactivation of one copy of the X chromosome in mammals that leads to major structural changes in the chromosome is initiated by the Xist lncRNA (Da Rocha and Heard 2017). LncRNAs can also mediate inter-chromosomal interactions, as an illustration, it has been reported that Firre lncRNA participates in the establishment of interactions between its own locus and two loci in another chromosomes (Hacisuleyman et al. 2014). Interestingly, DNase Hi-C data of human cell lines have shown that lncRNA promoters can interact with super-enhancers (Ma et al. 2014), while these interactions could be merely to regulate the transcription of lncRNAs, their existence raises the idea that lncRNAs might have specific functions within the multi-molecular assemblies formed by super-enhancers.\\

In congruence with their proposed roles in genome organization, whole lncRNA loci are located within anchors of large DNA loops formed in the inactive human X chromosome (Rao et al. 2014) and more extensively, it has been identified that lncRNAs are enriched at contact and loop domain boundaries in the genomes of mouse and human (Amaral et al. 2016; Tan et al. 2017). Interestingly, enrichment at boundaries is stronger for those lncRNAs transcribed from analogous genes in mouse and human syntenic regions (Amaral et al. 2016). However, lncRNAs enriched at contact domains coincide with chromatin marks characteristic of enhancers (Tan et al. 2017), which hinders the analysis of lncRNA functions. For instance, analyses of two lncRNAs, which appear to impact the expression of neighboring genes based on deletions of whole lncRNA loci, enabled to conclude that the functional regions can be narrowed to the enhancers contained within those loci, whereas the transcripts \textit{per se} are not necessary to regulate gene expression (Groff et al. 2016; Paralkar et al. 2016). Nonetheless, even if a lncRNA transcript does not have an apparent function, lncRNA loci can impact gene expression by enhancer hijacking. This mechanism has been observed in human cells, where the promoter of the PVT1 lncRNA competes with the \textit{MYC} promoter to interact with the enhancers overlapping the \textit{PVT1} locus. Although the lncRNA does not play a role in this interaction, it reflects the insulating activity of the promoter to block the expression of the \textit{MYC} oncogene (Cho et al. 2018a). Future analyses of lncRNAs will determine if more lncRNA loci can act through enhancer hijacking, as well as, the functions of lncRNAs enriched at contact domain boundaries, to assess if they could act independently of their overlapping enhancers.\\

	\section{Zebrafish as an ideal model to understand gene regulation}

As it can be noted from the previous sections, most of our understanding of gene regulation in vertebrates has been obtained by analyses of the mouse and human genomes. Comparison of the mechanisms contributing to the 3D genome organization throughout eukaryotes have shed light into important differences between the mechanisms operating in mammals and other eukaryotes (Rowley et al. 2017). Therefore, analyses of conservation of relevant regulatory elements in phylogenetically distant vertebrates from mammals are important to evaluate which mechanisms are conserved, at least throughout vertebrate evolution, and to identify potential novel regulators of gene expression.\\

Zebrafish is an ideal model to understand gene regulation because it counts with a well annotated genome reference (Howe et al. 2013), it enables the study of regulators \textit{in vivo} (Kang et al. 2016) and its genome can be easily engineered with genome editing techniques, such as CRISPR/Cas9 (Hwang et al. 2013; Auer et al. 2014). Also, given its external development and the possibility to analyze thousands of zebrafish embryos, zebrafish has been positioned as a main model organism to study development. Major emphasis has been put on understanding the transcriptional regulation during early zebrafish development. For instance, transcriptomic analyses have characterized the expression profiles of coding and noncoding RNAs (Vesterlund et al. 2011; Pauli et al. 2012; White et al. 2017), CAGE analyses have shown differential promoter usage in maternal and zygotic transcriptomes and identified a novel motif enriched at core promoters (Nepal et al. 2013), analyses of nucleosome positioning have determined that the canonical array of nucleosome free promoter regions emerges during zygotic genome activation (Zhang et al. 2014), and analyses of chromatin marks have identified the existence of bivalent promoters in embryonic pluripotent cells (Vastenhouw et al. 2010). More recently, lineage trajectories in whole zebrafish embryos have been described using single-cell RNA-seq, revealing the plasticity of fate specification and providing lineage markers that could be used for isolation of specific cell types (Wagner et al. 2018; Farrell et al. 2018).\\

Besides extensive annotations of zebrafish promoters used during development, annotations of stage-specific enhancers are also available (Aday et al. 2011; Bogdanović et al. 2012; Lee et al. 2015). Similarly to mammalian enhancers, zebrafish enhancers have low sequence conservation in general (Bogdanović et al. 2012; Lee et al. 2015). However, one disadvantage of these annotations is that cell and tissue specificity of the enhancers cannot be known \textit{a priori} given that they were generated using whole embryos. To overcome this limitation, identification of enhancers can be performed in specific cell types, as performed to identify  endothelial specific enhancers in embryos by analyzing accessible chromatin regions in cell populations labelled by a reporter gene (Quillien et al. 2017).\\

Importantly, super-enhancers are also regulators in zebrafish, but its particular features and conservation had not been determined. Super-enhancers were first characterized in this organism through analyses of a zebrafish melanoma model. These analyses showed that super-enhancers identified in cancer samples associate with genes that control neural fate, which expression is a hallmark of melanoma onset (Kaufman et al. 2016), suggesting that zebrafish super-enhancers are also involved in the establishment of cell identity. Also, it has been reported that super-enhancers identified in zebrafish embryos can interact even when they are located in different chromosomes (Kaaij et al. 2018).\\

Another well-described regulator of gene expression present in zebrafish is CTCF. Despite the third whole genome duplication that occurred in teleost fish, \textit{ctcf} is present in a single copy in the zebrafish genome and no paralogs have been identified (Pugacheva et al. 2006). \textit{ctcf} mRNA is maternally contributed and ubiquitously expressed in early development (Pugacheva et al. 2006; Carmona-Aldana et al. 2018). However, the roles of CTCF in zebrafish have not been precisely identified and current data is not conclusive regarding its essentiality. Morpholino-based assays targeting CTCF suggest that it is required for proper development and its knockdown has been associated to misregulation of certain genes (Delgado-Olguín et al. 2011; Rhodes et al. 2010; Marsman et al. 2014; Carmona-Aldana et al. 2018; Meier et al. 2018). Nevertheless, zebrafish morphants can have non-specific phenotypes (Kok et al. 2015), obscuring the identification of the direct effects of CTCF depletion. Surprisingly, a recent attempt to generate zebrafish CTCF mutants by the CRISPR/Cas9 system was unsuccessful and the authors reported high mortality rates in injected embryos, which was attributed to the presence of deletions in the open reading frame identified by analysis of pool of embryos. Of note, the authors also reported a higher mortality rate of uninjected embryos compared to mock injected embryos, which is contrary to the expected (Carmona-Aldana et al. 2018). Considering that Cas9 mRNA was used in the injection solution it is unlikely that all cells could have carried deletions in both alleles, which is confirmed by the identification of wild type alleles during the genotyping of pool of embryos (Carmona-Aldana et al. 2018). Therefore, the exact roles of CTCF during development and its essentiality are still unknown in zebrafish.\\

Hi-C and 4C data of zebrafish embryos combined with \textit{in silico} predictions of CTCF binding suggest that CTCF could have equivalent roles in the 3D genome organization to those described in mammals, including its role in the formation of DNA loops and contact domains (Tena et al. 2011; Gómez-Marín et al. 2015; Kaaij et al. 2018). Moreover, based on \textit{in silico} predictions it has been proposed that CTCF in zebrafish also has a preferential divergent orientation at contact domain boundaries (Gómez-Marín et al. 2015; Kaaij et al. 2018). Nevertheless, this observation has not been confirmed using genome-wide data of CTCF binding.\\

\section{Aims of the doctoral projects}

During the past ten years, the gene regulation field has been revolutionized by discoveries that have expanded the vision of one promoter controlled by \textit{cis}-regulators to a more complex vision in which networks of DNA, RNA and proteins interact in 3D space to achieve the establishment of expression programs. However, conservation of the regulators identified mainly through analyses in mammals has not be broadly investigated in additional vertebrate organisms. In order to evaluate if two of the regulators, considered to have central roles in gene regulation in mammals, are conserved or have similar functions in a distant vertebrate from mammals, I have focused on the characterization of super-enhancers and the analysis of CTCF binding in the zebrafish genome.\\

The genetic tools available for zebrafish and the availability of a good reference genome have in fact proved crucial to study super-enhancers and CTCF. First, for the analyses of super-enhancers, I focused on their identification to then proceed with a descriptive analysis of their characteristics, followed by the evaluation of their conservation in reference to those identified in mouse and human (Pérez-Rico et al. 2017). Second, to interrogate CTCF binding in the zebrafish genome, I raised a transgenic zebrafish line with a tagged version of CTCF. Then, I used this line to generate ChIP-seq libraries that enable to assess the distribution of CTCF relative to transcription units and contact domains (Pérez-Rico et al. in preparation).\\

This work aims to contribute to the analyses of gene expression in zebrafish and in a more broad sense, to the integrative evaluation of the functions of gene regulators throughout vertebrate evolution.\\



